nsq.h

```diff
@@ -23,18 +33,21 @@ struct NSQReader {
     struct NSQLookupdEndpoint *lookupd;
     struct ev_timer lookupd_poll_timer;
     struct ev_loop *loop;
+    struct NSQReaderCfg *cfg;
     void *httpc;
     void (*connect_callback)(struct NSQReader *rdr, struct NSQDConnection *conn);
     void (*close_callback)(struct NSQReader *rdr, struct NSQDConnection *conn);
     void (*msg_callback)(struct NSQReader *rdr, struct NSQDConnection *conn, struct NSQMessage *msg, void *ctx);
 };
 
 struct NSQReader *new_nsq_reader(struct ev_loop *loop, const char *topic, const char *channel, void *ctx,
+    struct NSQReaderCfg *cfg,
     void (*connect_callback)(struct NSQReader *rdr, struct NSQDConnection *conn),
     void (*close_callback)(struct NSQReader *rdr, struct NSQDConnection *conn),
     void (*msg_callback)(struct NSQReader *rdr, struct NSQDConnection *conn, struct NSQMessage *msg, void *ctx));
 void free_nsq_reader(struct NSQReader *rdr);
 int nsq_reader_connect_to_nsqd(struct NSQReader *rdr, const char *address, int port);
+int nsq_init_lookupd(struct NSQReader *rdr);
```

Comment:
I'm trying to mirror the API that exists in other client libraries like `pynsq` and `go-nsq`, we colloquially use the term "connect" despite it not _really_ connecting to `nsqlookupd`.

By calling it (presumably with an address), it will trigger the poll loop.

What do you think?